testsuite/integration/web/src/test/java/org/jboss/as/test/integration/web/formauth/FormAuthUnitTestCase.java

```diff
@@ -108,6 +98,8 @@ public static WebArchive deployment() {
         war.addAsWebResource(tccl.getResource(resourcesLocation + "restricted/error.html"), "restricted/error.html");
         war.addAsWebResource(tccl.getResource(resourcesLocation + "restricted/login.html"), "restricted/login.html");
 
+        war.as(ZipExporter.class).exportTo(new File("/home/darranl/tmp/Test.war"), true);
```

Comment:
I was investigating why this test was failing on lots of CI jobs and it never occurred to me that the path in the failure is clearly wrong for CI. :)

Reply:
Oops